addons/website/static/src/snippets/s_tabs_images/000.scss

```diff
@@ -3,9 +3,16 @@
         display: none;
     }
 
-    .s_tabs_nav_with_descriptions:where(.s_tabs_nav_vertical) :where(.o_nav_tabs_description) {
+    .s_tabs_nav_with_descriptions:where(.s_tabs_nav_vertical) :where(.o_nav_tabs_description),
+    .s_tabs_nav_with_descriptions_active:where(.s_tabs_nav_vertical) .active > :where(.o_nav_tabs_description) {
         @include media-breakpoint-up(lg) {
             display: block;
         }
     }
+
+    // Forces custom colors to be maintained independently of the color preset.
+    .s_tabs_pills_template .nav-link.active {
+        background: var(--nav-pills-link-active-bg) !important;
+        color: var(--nav-pills-link-active-color) !important;
+    }
```

Comment:
This seems to force the background to be broken.

That setup:
<img width="1411" height="487" alt="image" src="https://github.com/user-attachments/assets/a0276394-5d22-4947-9b3b-1f257492fc77" />

Suddenly becomes:
<img width="1351" height="495" alt="image" src="https://github.com/user-attachments/assets/808c28f9-04f5-416b-bfa9-66fa591fe531" />

after your commit (without the user touching the new options).